rpc/json.go

```diff
@@ -57,6 +57,40 @@ type jsonrpcMessage struct {
 	Result  json.RawMessage `json:"result,omitempty"`
 }
 
+type JsonRPCArray struct {
```

Comment:
I am surprised the node needs these. Please double check why these belong in the node.